src/module/actor/modifiers.ts

```diff
@@ -140,6 +140,13 @@ class ModifierPF2e implements RawModifier {
     notes: string;
     hideIfDisabled: boolean;
 
+    unidentified: boolean;
```

Comment:
Make this `identified`

Reply:
Why ? we have worked with `unidentified` in the effect.

Reply:
I agree with reonZ here. identified is the neutral state. Unidentified is the special state.

Reply:
Granted we can do `identified ??= true` during data preparation and that can work, though we'd need to also flip the default in effect. Not like null is a valid value.